src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use multiples of 4 for padding and margin values. Consider adjusting the padding to comply with this rule.